packages/create-react-app/index.js

```diff
@@ -163,9 +163,13 @@ function install(useYarn, dependencies, verbose, isOnline) {
       command = 'yarnpkg';
       args = [
         'add',
-        '--exact',
-        isOnline === false && '--offline'
```

Comment:
I've tried using a ternary operation here but they all failed, and i didn't want to apply a filter function after concat because i think it obscures the intent.